src/main/java/org/ossgang/commons/observables/DispatchingObservableValue.java

```diff
@@ -50,7 +52,7 @@ public Subscription subscribe(Observer<? super T> listener, SubscriptionOption..
         Set<SubscriptionOption> optionSet = new HashSet<>(Arrays.asList(options));
         Subscription subscription = super.subscribe(listener, options);
         if (optionSet.contains(FIRST_UPDATE)) {
-            Optional.ofNullable(lastValue.get()).ifPresent(listener::onValue);
+            Optional.ofNullable(lastValue.get()).ifPresent(value -> Maybe.attempt(() -> dispatch(listener::onValue, value).get()));
```

Comment:
I think this breaks another contract: FIRST_UPDATE is currently guaranteed to happen before subscribe() returns, on the subscribing thread.

Reply:
yes, this is what we discussed some time ago in the #42 issue. I think it's nice to dispatch all the values from the same thread pool and agree it should happen before the subscribe call returns :slightly_smiling_face: 

This way misbehaving observers will not affect the caller thread (very edge case.. but still possible) and the value propagation is not a special case anymore. Do you still agree Michi?

Reply:
OK, I see, I missed that you get() the future, so the first part is actually fine, and yes, I agree the "on the called thread" part we can probably just get rid of.

Thanks, fine for me :-)

Reply:
Perhaps Uncheckeds.uncheckedConsumer() could help making this a little more compact?

Reply:
Perfect! :slightly_smiling_face:

For the consumer I wanted to swallow that exception actually.. if the listener throws it's deflected.. if we get some of the usual Interrupt or Cancelled exception then we can potentially hurt the called thread.

Anyway, they're so rare that probably is not worth to "protect" anything and just let it crash since probably something else in the system is messed up :thinking:

Reply:
Yes, I would rather let it escape so it at least gets logged somewhere (hopefully).

Reply:
Cool so I merge :slightly_smiling_face: